server/lib/notifications.js

```diff
@@ -215,9 +215,61 @@ const notifyUpdateSubscribers = async activity => {
   const emailOpts = { from: activity.data.fromEmail };
   const update = await models.Update.findByPk(activity.data.update.id);
   const allUsers = await update.getUsersToNotify();
-  return notifySubscribers(allUsers, activity, emailOpts);
+  const modifiedActivity = replaceVideosByImagePreviews(activity);
+  return notifySubscribers(allUsers, modifiedActivity, emailOpts);
 };
 
+function replaceVideosByImagePreviews(activity) {
+  const iframePreviewRegex = /<iframe\s[^>]*src="([^"]+)"[^>]*><\/iframe>/gi;
```

Comment:
I'm not comfortable with us using regexps. This one won't cover all use cases (what if the regex has no closing tags? or something between the tags (i.e. `<iframe>Hello</iframe>`)? I'm also uncertain about the `.replace`, it's difficult to rely on this.

I'd like us to use a real HTML parser to achieve this. Thankfully, `sanitize-html` has just what we need: https://github.com/apostrophecms/sanitize-html#transformations. By using the `transformTags` option we'll be able to replace iframes by images safely and the code should be easier to understand.

Reply:
Thanks. I've used the transformations from `sanitize-html`. Let me know how it looks. I've enabled all content types we support except video embeds; 

https://github.com/opencollective/opencollective-api/blob/b7b4aef1c118a594255e1bab1ba4c4702fe117f4/server/lib/notifications.js#L245-L275

since we area dealing with a rich text editor text. 🤔